assets/Vagrantfile

```diff
@@ -3,6 +3,9 @@ $install_storageos = <<SCRIPT
 # Prerequisites
 sudo modprobe nbd nbds_max=1024
 
+# Activate UI
+export ALPHA_FEATURES=true
+
```

Comment:
Doesn't hurt to add this, but not needed.

Reply:
Reverted.